src/home.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+        <link rel="stylesheet" href="https://charcoal-ui.netlify.app/src/style/style.css"/>
+        <link rel="stylesheet" href="./style/nav-bar.css"/>
+        <link rel="stylesheet" href="./style/home.css"/>
+        <title>Ecommerce</title>
+    </head>
+    <body>
+        <nav class="ecom-nav-bar bg-white">
+            <div class="logo">
+                <div class="logo-text charcoal-black">MyWebsite</div>
+            </div>
+            <div class="nav-search-bar">
+                <input type="text" placeholder="Search" class="nav-search-input">
+                <span class="material-icons search-icon">search</span>
+            </div>
+            <div class="nav-bar-action">
+                <button class="btn white bg-charcoal-gray">Login</button>
+                <button class="btn btn-round bg-white badge-wrapper">
+                    <i class="material-icons">
+                        favorite_border
```

Comment:
Is it a class name or content ?

Reply:
it is google font for favorite icon with border

Reply:
yes thanks,  I got to know about material icon after this